tests/test_nginx_config_reloader.py

```diff
@@ -329,19 +329,22 @@ def test_that_handle_event_applies_config(self):
 
         self.kill.assert_called_once_with(42, signal.SIGHUP)
 
-    def test_that_handle_event_applies_config_if_no_notifier(self):
-        tm = self._get_nginx_config_reloader_instance(notifier=None)
+    def test_that_handle_event_clears_queue(self):
+        notifier = Mock()
+        notifier._eventq = deque(range(10))
+        tm = self._get_nginx_config_reloader_instance(notifier=notifier)
         tm.handle_event(Event('some_file'))
 
-        self.kill.assert_called_once_with(42, signal.SIGHUP)
+        self.assertEqual(notifier._eventq, [])
```

Comment:
Shouldn't it be an empty deque instead?

Reply:
An empty deque is equal to an empty list (it's still of type deque though)